de-lib/src/main/java/org/iplantc/de/theme/base/client/collaborators/GroupDisplayStrings.properties

```diff
@@ -1,3 +1,5 @@
+addGroup=Add List
+deleteGroup=Delete List
 deleteGroupConfirm=By deleting the collaborator list {0}, any data that has been shared with this list will immediately be un-shared.  Do you wish to continue deleting the collaborator list?
```

Comment:
To close the loop, we're thinking of implementing the following changes:
1) When a collaborator is removed from the default set of Collaborators, a warning will be displayed that tells the user that removing the collaborator will have no effect on existing permissions with that collaborator.
2) When a collaborator list is modified/deleted, the user will be presented with the option to either preserve the existing permissions or remove them.